### PR TITLE
Add token bridge guidance to the Link Token Contracts page.

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

File: docs/Developer Reference/link-token-contracts.md [hunk 1]
```diff
@@ -259,11 +258,11 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
...
 ### Rinkeby Testnet
 
-> 🚰Arbitrum Faucet on Rinkeby
+> 🚰 Arbitrum Faucet on Rinkeby
 >
 > Testnet LINK is available from https://faucets.chain.link/arbitrum-rinkeby
->
-> Supported bridge: https://bridge.arbitrum.io
+
+Testnet ETH is used to pay for transactions on the Arbitrum Rinkeby testnet. You can use the [Arbitrum Bridge](https://bridge.arbitrum.io/) to transfer testnet ETH and LINK from Ethereum Rinkeby to Arbitrum mainnet.
```
Comment (aelmanaa) [inline docs/Developer Reference/link-token-contracts.md:265]:
```suggestion
Testnet ETH is used to pay for transactions on the Arbitrum Rinkeby testnet. You can use the [Arbitrum Bridge](https://bridge.arbitrum.io/) to transfer testnet ETH and LINK from Ethereum Rinkeby to Arbitrum Rinkeby.
```

File: docs/Developer Reference/link-token-contracts.md [hunk 2]
```diff
@@ -201,10 +197,9 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 ### Fuji Testnet
 
-> 🚰Avax Faucet
+> 🚰 Avax Faucet
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Developer Reference/link-token-contracts.md:200]: No this is working as expected.

File: docs/Developer Reference/link-token-contracts.md [hunk 3]
```diff
@@ -349,6 +356,8 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 
 ### Mainnet
 
+GLMR is used to pay transaction fees on Moonbeam mainnet.
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Developer Reference/link-token-contracts.md:359]: Leaving this one unaddressed for now.

**Comment by dwightjl** [comment]
> Also:
> 
> * HECO: add a [bridge](https://www.hecochain.com/starsbridge/) ?

https://www.hecochain.com/starsbridge/ is whitelist only right now.

They also have these docs: https://docs.hecochain.com/#/hecobridge

Let's leave this as it is for now.

**Review by aelmanaa — APPROVED** [review]

**Pull request merged**